app/models/site_customization/content_block.rb

```diff
@@ -1,5 +1,5 @@
 class SiteCustomization::ContentBlock < ActiveRecord::Base
-  VALID_BLOCKS = %w(top_links footer)
+  VALID_BLOCKS = %w(top_links footer main_navigation_left main_navigation_right)
```

Comment:
What about switching to `subnavigation_left` and `subnavigation_right` to add consistency with `shared/_subnavigation.html.erb` name partial 😌

Reply:
I would agree. I noticed the partial names used in the other two. But I think again Devin was looking at the english wording that shows in the menu.

I think we can make the select option text be sent through i18n to get wording based on locale? That way the select list can be worded as organizations prefer.